frontend/src/views/WheresMyGene/components/Filters/components/Organism/index.tsx

```diff
@@ -15,7 +15,7 @@ import { selectOrganism } from "src/views/WheresMyGene/common/store/actions";
 import { Organism as IOrganism } from "src/views/WheresMyGene/common/types";
 import { StyledDropdown, Wrapper, Label } from "../common/style";
 
-const TEMP_ALLOW_NAME_LIST = ["Homo sapiens", "Mus musculus"];
+export const TEMP_ALLOW_NAME_LIST = ["Homo sapiens", "Mus musculus"];
```

Comment:
to prevent circular dependency, let's move this constant to `useAvailableOrganisms`!